nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/lifecycle/event/BulkLifeCycleChangeListener.java

```diff
@@ -131,11 +131,7 @@ protected void processTransition(Event event) {
                 transition = LifeCycleConstants.UNDELETE_TRANSITION;
                 targetState = ""; // unused
             }
-            ConfigurationService confService = Framework.getService(ConfigurationService.class);
-            boolean paginate = confService.isBooleanPropertyTrue(PAGINATE_GET_CHILDREN_PROPERTY);
-            long pageSize = paginate ? Long.parseLong(confService.getProperty(GET_CHILDREN_PAGE_SIZE_PROPERTY, "500"))
-                    : GET_CHILDREN_PAGINATION_DISABLED_FLAG;
```

Comment:
So `GET_CHILDREN_PAGINATION_DISABLED_FLAG` is effectively unused now? I think the constant can be removed. Maybe NXP-22197 (that introduced it) should be linked to this ticket as well.